cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -976,6 +983,32 @@ public void getServiceInstances(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Return the
+   */
+  @GET
+  @Path("/apps/{app-id}/services/{service-id}/active")
+  public void getServiceStatus(HttpRequest request, HttpResponder responder,
+                               @PathParam("namespace-id") String namespaceId,
+                               @PathParam("app-id") String appId,
+                               @PathParam("service-id") String serviceId) throws NotFoundException {
+    ProgramId programId = Ids.namespace(namespaceId).app(appId).service(serviceId);
+    ProgramStatus status = lifecycleService.getProgramStatus(programId);
+    if (status == ProgramStatus.STOPPED) {
+      responder.sendString(HttpResponseStatus.BAD_REQUEST, "Service is stopped. Please start it.");
```

Comment:
From the 503 explanation, I was mislead by this - `Generally, this is a temporary state`. So I thought if the service is stopped, then it is not a temporary state, until it is started. 

Regarding, restarting, I agree that might be confusing. But since we don't have a restart request in for non system services. So the user has to make two different calls?

I will change it to 503 since it looks like we don't get much benefit out of having that separate status code.